crates/l2/operator/l1_watcher.rs

```diff
@@ -1,3 +1,62 @@
-pub struct L1Watcher;
+use std::{cmp::min, time::Duration};
 
-pub async fn start_l1_watcher() {}
+use ethereum_types::{Address, H256, U256};
+use tokio::time::sleep;
+use tracing::debug;
+
+use crate::rpc::l1_rpc::L1Rpc;
+
+pub async fn start_l1_watcher() {
```

Comment:
Let's either add comments explaining what `0xe441CF0795aF14DdB9f7984Da85CD36DB1B8790d` and `0xe2ea736f80f92a510d75d1a96b5c1d5e5544283362b7acd97390d60ea1c7d149` are, or define them as constants.